### ISSUE TITLE
Cant install nvidia drivers

### ISSUE THREAD

**Issue by Robbier2713** [valinet/Win11DisableRoundedCorners#26]
when this is installed, the nvidia driver installer doesnt have permissions to restart the dwm.exe process so the install fails

**Comment by valinet** [comment]
Yeah, idk, this just replaces a file, I don't understand why that would affect whether some process is able to restart the other, but whatever...

This method is kind of deprecated, use [ExplorerPatcher](https://github.com/valinet/ExplorerPatcher) to achieve this (and more) or configure [ep_dwm](https://github.com/valinet/ep_dwm) standalone.

**Comment by Robbier2713** [comment]
> Yeah, idk, this just replaces a file, I don't understand why that would affect whether some process is able to restart the other, but whatever...
> 
> This method is kind of deprecated, use [ExplorerPatcher](https://github.com/valinet/ExplorerPatcher) to achieve this (and more) or configure [ep_dwm](https://github.com/valinet/ep_dwm) standalone.

wow thanks for the quick reply! i will seek the explorerpatcher route thanks!

**Comment by valinet** [comment]
No worries. If you just want to disable rounded corners, only configure ep_dwm. ExplorerPatcher includes many more other things, including ep_dwm built-in.
